pymc/distributions/multivariate.py

```diff
@@ -1517,74 +1515,57 @@ def make_node(self, rng, size, n, eta):
 
     @classmethod
     def rv_op(cls, n: int, eta, *, rng=None, size=None):
-        # We flatten the size to make operations easier, and then rebuild it
+        # HACK: normalize_size_param doesn't handle size=() properly
```

Comment:
It doesn't? Size=(), has a specific mean. It means no batch dimensions

Reply:
I forget why I wrote this but I believe my past self.

Reply:
rolled this back